icons/myanimelist.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>MyAnimeList icon</title><path d="M15.293 7.19c-.17 0-.307.001-.373.005-2.151.112-3.677.743-4.507 1.866-.82 1.103-.99 2.717-.498 4.714a17.62 17.62 0 001.135 3.034l1.66-1.087c.035-.033.026-.062-.066-.199-.26-.397-.498-.986-.67-1.686-.11-.438-.213-1.04-.186-1.063l2.823.012-.002 2.826 2.323.012-.021-6.314h-2.294l-.047 1.825-2.658-.014c-.036-.054-.015-.256.065-.507.267-.83.765-1.34 1.535-1.574.577-.232 3.763-.11 3.763-.11 0-.011-.116-.393-.258-.853-.246-.794-.26-.838-.335-.853a28.051 28.05 0 00-1.39-.034zm3.493.008v8.444l4.797.039.418-1.743-3.08-.074-.025-6.61zM.021 7.227l-.02 8.394h2.132v-5.214l2.022 2.513 2.023-2.454v5.155h2.09V7.227h-2.09L4.155 9.74 2.133 7.227z"/></svg>
```

Comment:
There's still some differences between this and the PNG unfortunately. You may find it easier to use a combination of the basic shapes and the pen tool to try and trace the icon - as we need them to be as accurate as possible. If you could take another look at this, and get it as close to the PNG as possible, I'll also be looking at this one over the next few days - as it seems to be a bit of a troublesome path!

![image](https://user-images.githubusercontent.com/1830867/104816870-c48c0400-5815-11eb-983c-ed4a97f4c95b.png)

Reply:
Just returning to this @ahmubashshir, I've generated a path manually that seems to be pretty close to the source, and at 3 decimals of precision. Feel free to compare it yourself and if it matches on your end, you're welcome to submit in the PR.

```svg
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>MyAnimeList icon</title><path d="M2.218 7.248l1.871 2.548 2.004-2.548h2.208v8.416H6.175v-5.181L4.15 12.858l-1.963-2.4-.044 5.206H0V7.248h2.218zm11.739 1.73h3.32l-.538-1.73s-.972-.095-1.81-.042a11.03 11.03 0 0 1-.163.01h-.003c-1.846.112-2.938.617-3.538 1.035l-.091.07C9.7 9.455 9.659 11.113 9.659 11.113s-.117 1.694.285 2.904c.401 1.211.64 2.01 1.108 2.793l1.72-1.104s-.453-.34-.906-2.299a3.878 3.878 0 0 1-.095-.615h2.827v2.872h2.334l-.025-6.33h-2.31l-.03 1.78h-2.675c.259-2.066 2.065-2.137 2.065-2.137zm6.932 4.889V7.222L18.763 7.2v8.464h4.81L24 13.888l-3.111-.02z"/></svg>
```